src/Union/README.md

```diff
@@ -0,0 +1,247 @@
+# INNOVATION SANDBOX
+
+## Installation
+
+The preferred method of installation is via Composer. Run the following command to install the package and add it as a requirement to your project's composer.json:
+
+```bash
+$ composer require enyata/innovation-sandbox
+```
+
+## Common Credentials
+
+Below is a list of required credentials.
+
+### sandbox_key
+
+This can be found in the innovation sandbox dashboard after signup. However`0ae0db703c04119b3db7a03d7f854c13` can be used for testing purposes.
+
+### host
+
+This argument is optional in all cases. Defaults to `https://sandboxapi.fsi.ng` if not found.
+
+## Token([options])
+
+You can generate a one-time access code also known as token for a UBN-MiServe transaction.
+
+### options
+
+The module accepts options as array of key-value.
+
+#### payload
+
+Request Body
+
+##### client_secret
+
+##### client_id
+
+##### grant_type
+
+##### Password
+
+##### username
+
+### InnovationSandbox\Union\Token\GenerateToken(credentials)
+
+Below is a sample with test data;
+
+```php
+<?php
+use InnovationSandbox\Union\Token;
+$instance1 = new Token();
+
+$this->instance1->GenerateToken(
+    '', 
+   'your sandbox_key', 
+    [
+       	'client_secret' => 'secret',
+        'client_id' => 'web01',
+        'grant_type' =>	'password',
+        'username' => 'ubnclient01',
+        'password' => 'w$777'
+    ]);
+```
+
+## Account Enquiry ([options])
+
+This operation provides basic account information of CASA or GL.
+
+### options
+
+The module accepts options as array of key-value.
+
+#### payload
+
+Request Body
+
+##### accountNumber
+
+Account number to be queried.
+
+##### accountType([optional])
+
+Account type. Default CASA.
+
+### InnovationSandbox\Union\Enquiry\Account(credentials)
+
+Below is a sample with test data;
+
+
+```php
+<?php
+use InnovationSandbox\Union\Enquiry;
+$instance1 = new Enquiry();
+
+$instance1->Account(
+    '', 
+   'your sandbox key', 
+    [
+       'accountNumber' =>'0000791200',
+       'accountType' =>'CASA',
+    ],
+    'your access_toke'
+    );
+```
+
+## Customer Enquiry([options])
+
+This operation provides basic customer information of CASA or GL.
+
+### options
+
+The module accepts options as array of key-value.
+
+#### payload
+
+Request Body
+
+##### accountNumber
+
+Account number to be queried.
+
+##### accountType([optional])
+
+Account type. Default CASA 
+
+### InnovationSandbox\Union\Enquiry\Customer(credentials)
+
+Below is a sample with test data;
+
+```php
+<?php
+use InnovationSandbox\Union\Enquiry;
+$instance1 = new Enquiry();
+
+$instance1->Customer(
+    '', 
+    'your sandbox key', 
+    [
+       'accountNumber' =>'+0000791200',
+       'accountType' =>'CASA',
+    ],
+    'access_token'
+    );
```

Comment:
Format properly here too and everywhere else necessary

Reply:
done